skribenten-web/frontend/src/Brevredigering/LetterEditor/actions/common.ts

```diff
@@ -65,24 +66,34 @@ export function deleteElements(
   }
 }
 
-export function newParagraph(...content: Content[]): ParagraphBlock {
+export function newParagraph(args: {
+  id?: Nullable<number>;
+  content: Content[];
+  deletedContent?: number[];
+}): ParagraphBlock {
   return {
     type: PARAGRAPH,
-    id: null,
+    id: args.id ?? null,
     editable: true,
-    deletedContent: [],
-    content,
+    deletedContent: args.deletedContent ?? [],
+    content: args.content,
   };
 }
 
 export function newLiteral(text: string): LiteralValue {
   return { type: LITERAL, id: null, text: "", editedText: text };
 }
 
+export const newVariable = (args: { id?: number; text: string; name?: string }): VariableValue => {
+  return { type: VARIABLE, id: args.id ?? -1, ...args };
```

Comment:
Vil ikke `...args` potensielt kunne overskrive `id` med `undefined` siden det er en gyldig type for `args.id`?

Reply:
Hmm, godt spørsmål. Det er så få elementer at jeg bare skriver dem inn direkte.